src/features/unified-share-modal/SharedLinkPermissionMenu.js

```diff
@@ -83,8 +84,7 @@ class SharedLinkPermissionMenu extends Component<Props> {
                             onClick={() => this.onChangePermissionLevel(level)}
                         >
                             <div>
-                                <strong>{permissionLevels[level].label}</strong>
-                                <small className="usm-menu-description"> {permissionLevels[level].description} </small>
+                                <span>{permissionLevels[level].label}</span>
```

Comment:
Only using the label instead of the description according to latest designs.

Reply:
Does this still have the same emphasis?

Reply:
This should not have the emphasis anymore according to design.